index.html

```diff
@@ -0,0 +1,397 @@
+                  </path>
+                  <image
+                    src="/assets/govuk-logotype-crown-0cdbde943be3a6518a5d7c00b607d134f4d5e3d0e8c68a1bfa2dc81f1f3edc4e.png"
+                    class="govuk-header__logotype-crown-fallback-image"></image>
+                </svg>
+                <span class="govuk-header__logotype-text">
+                  GOV.UK
+                </span>
+              </span>
+              <span class="govuk-header__product-name gem-c-header__product-name">
+                Account
+              </span>
+            </a>
+          </div>
+
+        </div>
+        <div class="govuk-header__content gem-c-header__content">
+
+        </div>
+      </div>
+    </div>
+  </header>
+
+  <div id="wrapper" class="govuk-width-container group">
+    <main id="content" role="main" class="group">
+      <div class="article-container">
+        <article role="article" class="group">
+          <div class="inner">
+            <h1>There is a problem with GOV.UK accounts</h1>
+            <p>Try again later</p>
+            <p><a class="govuk-link" href="https://www.gov.uk/transition">Go to the Brexit transition page</a> to find
+              out what you need to do to get ready.</p>
+            <br>
+            <pre>Status code: 503</pre>
+          </div>
+        </article>
+      </div>
+
+      <script type="text/javascript">
+        window.httpStatusCode = '503';
+      </script>
+    </main>
+
+  </div>
+
+
```

Comment:
I notice that the error pages in static contain a feedback form, is that something we're not including here on purpose?

Reply:
Yeah I think so. We've generally hijacked feedback to be separate from the rest of GOV.UK but as those are pages hosted within the app itself we've decided to remove links that go there too.

We may want to redirect folks to normal GOV.UK Feedback but I think we should add that whenever we would update this page to give more specific info on the nature of the outage. Then we could alert the feedback folks of a potential incoming deluge.